<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleWidget</class>
 <widget class="QWidget" name="RuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>60</number>
   </property>
   <property name="leftMargin">
    <number>70</number>
   </property>
   <property name="topMargin">
    <number>70</number>
   </property>
   <property name="rightMargin">
    <number>70</number>
   </property>
   <property name="bottomMargin">
    <number>90</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="rule" stretch="0,0,0">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>150</number>
     </property>
     <property name="topMargin">
      <number>90</number>
     </property>
     <property name="rightMargin">
      <number>150</number>
     </property>
     <item>
      <widget class="QLabel" name="ruleContext">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>140</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;規則：&lt;/p&gt;&lt;p&gt;按下開始遊戲後，請按下正確選項以作答。&lt;/p&gt;&lt;p&gt;按下錯誤選項將自動顯示正確選項。&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="quantity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>題目庫共有n條，每次開始遊戲系統會隨機抽取k條顯示。</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="returnAligner">
       <item>
        <widget class="QPushButton" name="returnButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft JhengHei</family>
           <pointsize>21</pointsize>
          </font>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
